lib/App/MintTag.pm

```diff
@@ -493,6 +495,41 @@ sub _maybe_delete_source_branches ($self, $step) {
   }
 }
 
+sub _maybe_cleanup_tags ($self, $step) {
+  return unless $step->push_tag_to && $step->cleanup_tag_days;
+
+  my $remote = $step->push_tag_to;
+  my $prefix = $step->tag_prefix;
+
+  try {
+    my $tag_spec = "refs/tags/${prefix}*";
+    my $tag_list = run_git('for-each-ref', "--format=%(refname:short)\t%(creatordate:unix)", $tag_spec);
+
+    my @tags = map {; my ($tag, $ts) = split /\t/; { ts => $ts, 'tag' => $tag} }
```

Comment:
nit: I'd reflow this for readability

```
my @tags = map {;
  my ($tag, $ts) = split /\t/;
  { ts => $ts, 'tag' => $tag}
}  split /\n/, $tag_list;
```

Or at least indent the `split /\n/, $tag_list;` so it's clear it belongs to the code above it